src/platforms/common/configuration/options.mdx

```diff
@@ -600,6 +600,10 @@ Options used to configure the transport. This is an object with the following po
 
 </ConfigKey>
 
+<ConfigKey name="proxy" supported={["native"]}>
+Configures an HTTP proxy for outgoing requests to the backend (i.e. HTTPS requests tunneled through that proxy). The Native SDK doesn't pick up the `http_proxy` environment variable. You can find more details on the [transport page](/platforms/native/configuration/transports/#using-an-http-proxy).
+</ConfigKey>
```

Comment:
```suggestion
<ConfigKey name="proxy" supported={["native"]}>
Configures an HTTP proxy for outgoing requests to the backend (HTTPS requests tunneled through that proxy, for example). The Native SDK doesn't pick up the `http_proxy` environment variable. You can find more details on the [transport page](/platforms/native/configuration/transports/#using-an-http-proxy).
</ConfigKey>
```

Reply:
In this case, this is meant as "i.e.," and not "e.g.,": "HTTPS requests tunneled through that proxy" this is not one example of many, but a more detailed description of what "outgoing requests to the backend are" and how they transferred.

I understand you want to eliminate "i.e." and friends, but I would leave ", for example" also out in this case.

Reply:
Fair point! We try to steer clear of Latin abbreviations in the docs. I will update.